crypto/data_formats.go

```diff
@@ -53,3 +55,66 @@ func HashLogRoot(root trillian.SignedLogRoot) ([]byte, error) {
 	}
 	return hash[:], nil
 }
+
+// LogRootV1 contains the fields verified by SignedLogRoot
+type LogRootV1 struct {
+	Version        uint16
+	TreeSize       uint64
+	RootHash       []byte `tls:"minlen:0,maxlen:128"`
+	TimestampNanos uint64
+	Revision       uint64
+	Metadata       []byte `tls:"minlen:0,maxlen:65535"`
+}
```

Comment:
Do we need `LogID` in the structure too?

Reply:
We discussed including the `LogID` in the struct, but decided against it in order to prevent a malicious log from falsely representing it's `LogID`. Clients should derive the LogID from the valid signature associated with `SignedLogRoot`